app/assets/stylesheets/sidebar_admin.css.scss

```diff
@@ -34,3 +34,7 @@
     display: inline-block;
   }
 }
+
+h3.sidebar_title {
+  font-family: monospace;
```

Comment:
Hi, I'm wondering what this was done for.  Is this not something we already have?  Please explain.